honestcash-v2/src/app/modules/editor/services/editor.service.ts

```diff
@@ -34,7 +34,7 @@ export const API_ENDPOINTS = {
   saveDraft: (p: Post) => `/v2/draft/${p.id}/body`,
   publishPost: (p: Post) => `/v2/draft/${p.id}/publish`,
   uploadImage: () => `/upload/image`,
-  uploadRemoteImage: () => `/v2/upload/image/remote`
+  uploadRemoteImage: () => `/upload/image/remote`
 };
 
 export enum STORY_PROPERTIES {
```

Comment:
- We already have a model "Post"
- The service/components files should have one export per convention